docs/docs/guides/compatibility.mdx

```diff
@@ -0,0 +1,29 @@
+---
+id: compatibility
+title: Compatibility
+sidebar_label: Compatibility
+---
+
+# Compatibility Table
+
+In order to provide solid, state-of-the arts solutions we often use brand new React Native features,
+that are being introduced all the time. Thats why applications with mismatched Reanimated and React Native
+versions may show unexpected behavior or even fail to build at all. The table below contains official list
+of supported React Native versions.
+
+#### Supported React Native versions - New Architecture (Fabric)
+
+| Reanimated version(s)       | The oldest supported React Native | The newest supported React Native |
+| --------------------------- | --------------------------------- | --------------------------------- |
+| from **3.1.x** to **3.5.x** | 0.72                              | 0.72                              |
+
+#### Supported React Native versions - Old Architecture (Paper)
+
+| Reanimated version(s)         | The oldest supported React Native | The oldest supported React Native |
+| ----------------------------- | --------------------------------- | --------------------------------- |
+| from **3.1.x** to **3.5.x**   | 0.72                              | 0.72                              |
+| from **3.4.x** to **3.5.x**   | 0.66                              | 0.72                              |
+| from **3.1.x** to **3.3.x**   | 0.63                              | 0.72                              |
```

Comment:
1. These intervals shouldn't overlap
2. I believe 3.5.x supports RN 0.70 as well
3. What about 3.0.x?

Reply:
1 & 2 - yes, it is row of fabric requirements
3 - I'll add 3.0.x